polaris/hub/client.py

```diff
@@ -229,44 +215,8 @@ def request(self, method, url, withhold_token=False, auth=httpx.USE_CLIENT_DEFAU
             if isinstance(error, httpx.HTTPStatusError) and error.response.status_code != 401:
                 raise
             raise PolarisUnauthorizedError from error
-        return response
-
-    # =========================
-    #     API Endpoints
-    # =========================
-
-    @property
-    def user_info(self) -> dict:
-        """Get information about the currently logged in user through the OAuth2 User Info flow."""
-
-        # NOTE (cwognum): We override the default `auth` and `headers` argument, since
-        #  the defaults trigger a 530 error (Cloudflare) due to the header ordering.
-        #  Because of this, we also have to copy some code from the base `request` method to
-        #  make auto-refresh a token if needed. For more info, see: https://stackoverflow.com/a/62687390
-
-        try:
-            if self.token is None or not self.ensure_active_token(self.token):
-                raise PolarisUnauthorizedError
-        except OAuthError:
-            raise PolarisUnauthorizedError
-
-        if self._user_info is None:
-            user_info = self.get(
-                self.settings.user_info_url,
-                auth=None,  # type: ignore
-                headers={"authorization": f"Bearer {self.token['access_token']}"},
-            )
-            user_info.raise_for_status()
-            self._user_info = user_info.json()
-
-        return self._user_info
-
-    @property
-    def user_as_owner(self) -> HubOwner:
-        """Easily get the currently logged-in user a `HubOwner` instance."""
-        return HubOwner(user_id=self.user_info["user_id"], slug=self.user_info["username"])
 
-    def login(self, overwrite: bool = False, auto_open_browser: bool = True):
+    def interactive_login(self, overwrite: bool = False, auto_open_browser: bool = True):
```

Comment:
**Documentation**: Although it is rare, we do have some places where we also show `PolarisHubClient().login()`. For example in [this tutorial](https://polaris-hub.github.io/polaris/stable/tutorials/basics.html) or in some of the tutorials we've given, e.g. at the ML4DD hackathon. Let's at least update the tutorial, but I would keep `login()` with a deprecation warning for the time being as well.

Reply:
Good point! I'll revert the name change.